ckan/lib/search/common.py

```diff
@@ -65,6 +66,16 @@ def is_available():
 
 def make_connection(decode_dates=True):
     solr_url, solr_user, solr_password = SolrSettings.get()
+
+    if solr_url and solr_user:
+        # Rebuild the URL with the username/password
+        protocol = re.search('(http(?:s)?)://', solr_url).group()
```

Comment:
should this say `.group(1)`?

Reply:
Hi @wardi, this is intentionally capturing the whole match:

```python
>>> import re
>>> solr_url="https://my.thing.com:8000"
>>> protocol = re.search('(http(?:s)?)://', solr_url).group()
>>> protocol
'https://'
```

However, I've just noticed that I didn't need that capture group, so I'll remove that now.